src/components/DialoguesWithoutCuratorButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { ChatBubble } from '@mui/icons-material';
+import { Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+export const DialoguesWithoutCuratorButton = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const onClick = () => {
+    const statusNew: TicketStatus = 'New';
+    searchParams.set(TicketSearchParam.Status, statusNew);
+    searchParams.delete(TicketSearchParam.Assignee);
+    history.push({ search: searchParams.toString() });
+  };
+
+  return (
+    <Button
+      fullWidth
+      size="large"
+      variant="outlined"
+      color="primary"
+      sx={{ textTransform: 'none' }}
+      startIcon={<ChatBubble />}
+      onClick={() => onClick()}
+      title={t('tasks.dialogues_without_curator')}
+      data-testid={dataTestId.tasksPage.dialoguesWithoutCuratorButton}>
+      <Typography noWrap>{t('tasks.dialogues_without_curator')}</Typography>
```

Comment:
Typography kan være skummelt å ha inni en Button, for da mister man all innebygd styling av knappetekst. bl.a. at teksten bytter farge om knappen er `contained`. Kan erstatte `Typography` med `<Box component="span">` elns, men jeg trooor det skal virke likt å sette no-wrap i sx til Button 🤔

Reply:
Godt tips! `whiteSpace: 'nowrap'` gjorde susen 👍